cli/tests/integrations/test_marathon.py

```diff
@@ -203,6 +203,17 @@ def test_remove_app():
     _list_apps()
 
 
+def test_add_multiapp():
+    with _zero_instance_multiapp():
+        _list_apps(['zero-instance-app1', 'zero-instance-app2'])
+
```

Comment:
Sorry, I suck at tests, can you explain what you mean by negative case? I went through the existing list of tests and thought that the others like e.g. bad json or adding multiple existing apps are already covered by the existing tests since they are using the same method and technically all apps (single and multi) are now multiple apps. Just that a single app ends up being a list with just one element in it. If that makes sense.